### DISCUSSION TITLE
How to control lines of post content displayed in the post listing page?

### DISCUSSION THREAD

**Discussion by Eric-Miao** [alshedivat/al-folio#1811] [Q&A] [answered]
I wonder 

1. if there is a way to control how many lines of content of a post being displayed in the post listing page 
2. how to add tag/category and describtions to the posts that are retrived from my external RSS feed link.

Thank you in advance!

**Comment by george-gca** [comment, marked as answer]
1. Actually what is shown there is not the content of the post, but the content of the `description` that is added in the post front matter.
2. This has to be implemented in `_plugins/external-posts.rb` I believe

**Reply by Eric-Miao** [reply]
Thank you!
